pysmt/test/test_regressions.py

```diff
@@ -289,6 +289,16 @@ def test_parse_define_fun_bind(self):
         buffer_ = cStringIO(smtlib_input)
         parser.get_script(buffer_)
 
+    def test_parse_bvx_var(self):
+        smtlib_input = """
+        (declare-fun bv1 () (_ BitVec 8))
+        (assert (bvult (_ bv0 8) (bvudiv (bvmul (bvadd bv1 (_ bv1 8)) (_ bv5 8)) (_ bv5 8))))
+        (check-sat)"""
+        parser = SmtLibParser()
+        buffer_ = cStringIO(smtlib_input)
+        parser.get_script(buffer_)
+
```

Comment:
This is only testing that the code does not crash. We can do much better, and actually test that we obtain the expression defined above.